eclair-core/src/main/scala/fr/acinq/eclair/channel/Channel.scala

```diff
@@ -815,7 +813,7 @@ class Channel(val nodeParams: NodeParams, val wallet: EclairWallet, remoteNodeId
       else if (Commitments.localHasUnsignedOutgoingHtlcs(d.commitments))
       // TODO: simplistic behavior, we could also sign-then-close
       handleCommandError(CannotCloseWithUnsignedOutgoingHtlcs(d.channelId), c)
-      else if (!Closing.isValidFinalScriptPubkey(localScriptPubKey))
+        else if (!Closing.isValidFinalScriptPubkey(localScriptPubKey))
```

Comment:
```suggestion
      else if (!Closing.isValidFinalScriptPubkey(localScriptPubKey))
```

Reply:
Fixed in https://github.com/ACINQ/eclair/pull/1473/commits/9037da2978266fb0de40bf13c272d0024dfedcea